src/background/custom-filters.js

```diff
@@ -209,29 +210,34 @@ async function update(text, { trustedScriptlets }) {
   return result;
 }
 
-observe('customFilters', async ({ enabled, trustedScriptlets }, lastValue) => {
-  // Background startup
-  if (!lastValue) {
-    // If custom filters are disabled, we don't care if engine was reloaded
-    // as custom filters should be empty
-    if (!enabled) return;
+observer.addListener(
+  'customFilters',
+  async ({ enabled, trustedScriptlets }, lastValue) => {
```

Comment:
maybe you can deconstruct the first argument in a function body - that way we may fit the line length limit and the rest of the function will require no indentation changes.